client/my-sites/subscribers/main.tsx

```diff
@@ -181,26 +181,25 @@ const SubscribersPage = ( {
 			sortTermChanged={ sortTermChanged }
 		>
 			<QueryMembershipsSettings siteId={ siteId ?? 0 } source="calypso" />
-			<Main
-				wideLayout
-				className={ `subscribers${
-					isEnabled( 'subscribers-dataviews' ) ? ' subscribers--dataviews' : ''
-				}` }
-			>
+			<Main wideLayout className="subscribers">
 				<DocumentHead title={ translate( 'Subscribers' ) } />
 
-				<SubscribersHeader
-					selectedSiteId={ selectedSite?.ID }
-					disableCta={ isUnverified || isStagingSite }
-				/>
+				{ ! isEnabled( 'subscribers-dataviews' ) && (
```

Comment:
Hide the header when flagged, because it's living inside the DataViews components.